yarn.lock

```diff
@@ -0,0 +1,14242 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This should be added to .gitignore and deleted from the repo itself as we are using npm in prod